salt/master.py

```diff
@@ -1038,7 +1035,7 @@ def __bind(self):
             pass
 
     @salt.ext.tornado.gen.coroutine
-    def _handle_payload(self, payload):
+    def _handle_payload(self, payload, *optional_message_properties):
```

Comment:
**question** would keyword args be better here? (i.e. `**` instead of `*`. )

Reply:
(looking through some of the uses later, :100: prefer keyword expansion)

Reply:
Done.